tests/testit/test-cran.R

```diff
@@ -0,0 +1,43 @@
+library(testit)
+
+assert("cran_package_dates() needs the name of a maintainer", {
+  has_error(cran_pkg_dates(maintainer = NULL))
```

Comment:
This is basically testing if `grep(NULL, x)` throws an error, so I feel it is not very meaningful as a test for `cran_package_dates()`.

BTW, this function requires internet connection, so I'd rather not test it on CRAN, but only test it on CI servers like Github Actions. You may see `test-travis.R` here for an example on how to split the tests and run them on different platforms: https://github.com/yihui/tinytex/tree/master/tests